config.json

```diff
@@ -14,22 +14,38 @@
             "collection": "bitcannon"
         }
     },
+    "torrents": {
+        "whitelist": {
+            "enabled": false,
+            "categories": []
+        },
+        "blacklist": {
+            "enabled": false,
+            "categories": []
+        }
+    },
     "session": {
         "secret": "asdasdasdasd43jn5kj2n43k"
```

Comment:
This should be "" as default and be created on first run otherwise all installs will be using the same session secret. I can't remember if we do that or not.